packages/uswds-core/src/styles/mixins/utilities/_transition.scss

```
@@ -0,0 +1,20 @@
+@use "sass:meta";
+@use "sass:string";
+@use "../../variables/project-easing" as *;
+
+// Outputs transition with project default easing and media query to disable inessential animation
+// when reduced motion is preferred.
+
```

SASSDoc style comments for mixins and functions would be really helpful.

```suggestion
/// Transitions with default easing and option to disable inessential
/// animations when reduced motion is preferred.
///
/// @param {String} $prop - Property to transition (e.g., 'top', 'opacity', 'all').
/// @param {String | Variable} $easing [$project-easing] - Easing function.
/// @param {Boolean} $essential [false] - Toggle essential transitions.
///
/// Example usage:
/// @include u-transition('top');
/// @include u-transition('opacity', 'ease-in-out');
/// @include u-transition('all', cubic-bezier(0.17, 0.67, 0.83, 0.67), true);
```

---

> SASSDoc style comments for mixins and functions would be really helpful.

Looks good 👍 Added in fdac9048d.